app/Http/Controllers/DepartmentsController.php

```diff
@@ -171,9 +175,35 @@ public function update(ImageUploadRequest $request, $id)
         $department = $request->handleImages($department);
 
         if ($department->save()) {
+
             return redirect()->route('departments.index')->with('success', trans('admin/departments/message.update.success'));
         }
 
         return redirect()->back()->withInput()->withErrors($department->getErrors());
     }
+
+    public function passwordSave(Request $request)
```

Comment:
Why is there a password save method in the ImageUploadRequest? Passwords and images are not related AFAIK.

Reply:
Wtf..I dont recall every even being in that file. fixing that now